### DISCUSSION TITLE
Social0 Issues and Planning

### DISCUSSION THREAD

**Discussion by jkbhagatio** [SainsburyWellcomeCentre/aeon_experiments#400] [General]
We're currently running social experiments and have come across a number of issues and are having minor internal disagreements about the best way to proceed, so we want to lay everything out here ahead of a meeting on Wed Nov 29 2024.

## Main Issues

1. Aeon4 PC Crashes
2. Experiment Schedules / Timelines
3. Environment / "Task" Logic Questions

### 1. Aeon4 PC Crashes

Over the past two weeks we've been getting repeated crashes on our Aeon4 PC. Initially we had planned to use Aeon3 and Aeon4 in parallel, but these crashes will potentially mean we can only proceed with Aeon3 for the near-term future. In regards to this, we should agree on what to do about Aeon4 and new experiment timelines. Which brings us to:

### 2. Experiment Schedules / Timeline

Currently we are running Social experiments on what I view as "test" animals - old mice >20 weeks. While we all agree that it's important to test the experiment workflows on these animals, I think there is some disagreement on how useful the data will be from these animals and how long we should run experiments on them. My preference would be to start experiments asap with the younger mice around the age we had initially targed (~8 weeks), so we should discuss what we actually want to get from these test mice. 

Furthermore, the runtime of current experiments will dictate the schedule in the near future. I will be in London all of December and January so I can potentially run experiments throughout the holidays, and I would like to discuss an experiment schedule during this time.

### 3. Environment / Task Logic Questions

There are currently some ongoing discussions regarding the Block structure of the current Social experiment workflow: these concern how blocks should start and end, and patch behavior within blocks. 

This is discussed a bit starting from [this comment](https://github.com/SainsburyWellcomeCentre/aeon_experiments/discussions/138#discussioncomment-7659778) in the "Social Experiments Planning & Timeline Doc" discussion. 

Because there are a few nuances here that may make it a little difficult to ensure everyone is on the same page just via text, I think it would be useful to discuss this and go through some example cases to make sure we are indeed all on the same page.

**Comment by Dario55** [comment]
1. Aeon4 has been fixed
2. The earliest we can start experiments on the new cohort is Nov 30th. The current experiment with the old mice is schedule to terminate around Dec 5th/6th. Since we have never run a social experiment back to front with the current workflow I would strongly suggest to get it to the end. We have already identified a number of issues with the workflow, setup, experiment logic that had to be fixed and we did not foresee before (as it is normal for the first time an experiment is run). I would be very surprised if postponing the experiment of a week for the new mice will change the results (if anything it will give them few more days to recover from the tattoo). 
in term of analysing data: sure 5 month old mice could have a dramatically different foraging and social behaviors than 2 month old mice (not sure if there are evidence for this). Since by the time the new cohort is ready to run we will be 2/3 in of the experiment, I would just finish it and we will have the data.



3. Since changing the implementation is quite complex I agree with Jeff comment in the other GitHub issue of leaving as it is  at the moment. Happy to discuss if changes will have to be made in the future. We can also look at the data to see if it actually matter.

**Comment by jkbhagatio** [comment]
Following discussion today, we've decided on attempting to carry out the following experiments in December (I would call these Social0.2.0):

Two male mice will run individually in Aeon3 and Aeon4 during the same days, for a total of 3 days. Afterwards, they will be placed together in Aeon4. At this time, two female mice will be placed together in Aeon3. Both arenas will thereafter run these social experiments for the rest of the month. 

These experiments will share the following:
  - Tube tests will be carried out on both pairs before they enter an arena together.
  - The possible combination of sample distributions for the three patches will be: HHH, LLL, every combo of H, L, M
  - The block durations will be sampled uniformly within a range of 1-3 h
  - On a block change, the distributions for all patches are immediately resampled (instead of each patch individually getting assigned a distribution at the time of the first following pellet delivery) 

We still need to decide on daily plots / viz.

The most pressing change to the experiment workflow is #404 

@jerlich @Dario55 @glopesdev can you confirm, and add anything I've left out?

**Reply by jerlich** [reply]
I suggested that the first order analysis can be based on RFID readers, looking at the 3x3 joint occupancy distribution of [nest, arena, patch].  We will have the "solo" occupancy distributions (for the males), so we can check if the joint distribution has any conditional dependency.

**Reply by Dario55** [reply]
Thank you for the meeting yesterday. I agree with most of the proposed changes and there are a couple of points I think can  even be improved further.

Summary (if you want to skip the long explanation below): My advice would be to run same experiment in both arenas with the structure mouse A ~3 days, mouse B ~3 days, 1 day gap; both mice ~3-4 weeks, and to use males everywhere (males vs females comparison is very interesting, just I don’t think it’s the most efficient thing to start with it). Below the rational of my concern: 

In natural behaviors, inter-subject variability is even higher than the one that  can be observed in highly trained mice  doing 2AFC task. For escape, we never dare to conclude anything from a  pilot, before we do at least 6-8 mice. However In the current experimental design we are starting with 4 conditions. Males vs female, single mouse sessions at the start vs no single sessions at the start. Each of them will run for about a month, so you can easily do the maths and see how long it will take to get enough data to make any valuable comparison between conditions. 

I would strongly recommend to start the same experiment in the two arenas and run that in parallel. This will give us n of 8 mice and 4 experiments with the same condition in 2 months, and would allow to have enough data to hopefully see some pattern.

In term of which experiment to run:
1)I would vote to go with males (given the absence o f constrained space the fights would not be as severe as in the home cage. Additionally, if we see this happening we can add on the fly an extra nest (with no scale). Jeff already mentioned anecdotal evidence of differences in the behavior, so likely we won’t be able to poll data between males and females.

2)I would vote to have individual sessions at the start for both arenas. Besides the importance of spotting individual behavioral patterns we discussed yesterday, this is also crucial for the analysis about coforaging. As far as I understood the idea is to look at the probability of individual foraging of each mouse, and see if the probability of coforaging is different than what you would expect by chance from multiplying each mouse probability. My question is how you would know the probability of individual foraging of a mouse in these environment if you don’t have single mouse data?

This is also in line with the discussion we had a while back  of focusing effort to get some scientific results before the next SAB.
If we aim to achieve this, giving up to some of the exploratory nature of the foraging project and channeling energy on specific experiments I feel may help lot. 

Hope this helps and sorry for being a pain in the neck.

**Reply by jerlich** [reply]
I am fine with Dario's suggestions.
